viz.yaml

```diff
@@ -264,10 +264,13 @@ publish:
       line1: >-
         Hurricane Matthew approached the southeastern U.S. coast on October 7, 2016.
         In the map above, the hurricane’s impact on precipitation and streamflow are 
-        shown. Discharge (cubic feet per second) 
-        at US Geological Survey gaging stations within ~150 km of the eye of the hurrican is shown 
-        through time (right panel). Gages shown do not include the US Geological Survey 
-        Short-Term Network gages deployed to capture more detailed effects of the hurricane (
+        shown. Normalized discharge (cubic feet per second) 
+        at US Geological Survey gaging stations within ~150 km of the eye of the hurricane is shown 
+		in the right panel. Variation in the shape of the hydrographs (right panel) 
+		is due to stream size, storm-surge, reservoir closures, and other local conditions, 
+		which can impact the effect of precipitation on flow. Gages shown do not 
+		include the US Geological Survey Short-Term Network gages deployed to capture 
+		more detailed effects of the hurricane (
```

Comment:
I can merge and take out the indents if you want. I'm not sure if that ending ( is something critical for our formatting or a mistake either.